src/main/java/no/nav/veilarbperson/client/difi/DifiCient.java

```diff
@@ -0,0 +1,11 @@
+package no.nav.veilarbperson.client.difi;
+
+import no.nav.common.health.HealthCheck;
+
+public interface DifiCient extends HealthCheck {
+    HarLoggetInnRespons harLoggetInnSiste18mnd(String fnr);
+
+    static String getDifiUrl() {
```

Comment:
Trenger ikke å være en del av interfacet. Blir heller en detalj for implementasjonen av interfacet